components/script/dom/xrview.rs

```diff
@@ -24,17 +25,24 @@ pub struct XRView {
     proj: Heap<*mut JSObject>,
     #[ignore_malloc_size_of = "mozjs"]
     view: Heap<*mut JSObject>,
+    viewsrc: Vec<f32>,
```

Comment:
let's call this `proj_array`, here and elsewhere in the code

```suggestion
    proj_array: Vec<f32>,
```